Readme.md

```diff
@@ -1,18 +1,24 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/132128333/23.1.6%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4483)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# How To: Implement a custom security operation that can be permitted at the type level
-<!-- run online -->
-**[[Run Online]](https://codecentral.devexpress.com/e4483/)**
-<!-- run online end -->
 
+# XAF - How to implement a custom security operation that can be permitted at the type level
 
-<p>In the XAF Security System, the following operations can be permitted for a specific type by default: <em>Read</em>, <em>Write</em>, <em>Create</em>, <em>Delete </em>and <em>Navigate</em>. In this topic, an implementation of the additional <em>Export </em>operation is shown. The customization of the <strong>ExportController </strong>which should consider whether or not the <em>Export </em>operation is allowed for the currently displayed object type is also demonstrated.</p>
-<p><img src="https://raw.githubusercontent.com/DevExpress-Examples/how-to-implement-a-custom-security-operation-that-can-be-permitted-at-the-type-level-e4483/16.1.4+/media/b0eb14fb-5014-11e6-80bf-00155d62480c.png"></p>
-<p>The complete description is available in the <a href="http://help.devexpress.com/#Xaf/CustomDocument3472"><u>online documentation</u></a>.</p>
+This example shows how to create a custom operation permission that allows you to prohibit the export operation when using Security System.
```

Comment:
When using what Security System? It needs an article. Any security system or a specific one?

Reply:
```suggestion
This example shows how to create a custom operation permission that allows you to prohibit the export operation when using the Security System.
```